launcher/tests/test_updater.py

```diff
@@ -1,9 +1,9 @@
-import imp
 import json
 import os
 import pytest
 import subprocess
 from datetime import datetime
+from importlib.machinery import SourceFileLoader
```

Comment:
Using `importlib` instead of `imp` here helps us to avoid the deprecation warning for `imp`.